gamehub-android/game/src/main/java/com/farsitel/bazaar/game/constants/Constant.java

```diff
@@ -1,11 +1,23 @@
 package com.farsitel.bazaar.game.constants;
 
 public class Constant {
-    public static final int MINIMUM_BAZAAR_VERSION = 1700000;
+    public static final int REQUIRED_BAZAAR_EVENT_FOR_TOURNAMENT = 1700000;
+    public static final int REQUIRED_BAZAAR_VERSION_FOR_EVENT = 1700000; // todo: update this to the released version
 
     public static final String BAZAAR_PACKAGE_NAME = "com.farsitel.bazaar";
+    public static final String BAZAAR_GAME_PACKAGE_NAME = "com.farsitel.bazaar.Game.BIND";
     public static final String BAZAAR_LOGIN_URL = "bazaar://login";
     public static final String BAZAAR_INSTALL_URL = "https://cafebazaar.ir/install";
     public static final String BAZAAR_DETAILS_URL = "bazaar://details?id=com.farsitel.bazaar";
     public static final String BAZAAR_TOURNAMENT_URL = "bazaar://tournament_leaderboard?package_name=";
+    public static final String CONNECT_TO_SERVICE_FIRST = "Connect to service first!";
+    public static final String SERVICE_IS_CONNECTED = "GameHub service is connected.";
+    public static final String SERVICE_IS_DISCONNECTED = "GameHub service is disconnected.";
+    public static final String SERVICE_IS_STARTED = "GameHub service is started.";
+    public static final String BROADCAST_IS_CREATED = "GameHub broadcast is created.";
+    public static final String GET_RANKING_NEEDS_UPDATE_BAZAAR = "Getting Ranking-data needs updating to last version of Bazaar!";
+    public static final String INSTALL_BAZAAR = "Install Bazaar to support GameHub!";
+    public static final String UPDATE_BAZAAR = "Install new version of Bazaar to support GameHub!";
+    public static final String TOURNAMENT_RANKING_IS_SHOWN = "The last tournament ranking table is shown";
+    public static final String LOGIN_TO_BAZAAR_FIRST = "Login to Bazaar first!";
```

Comment:
They are combined of anything, categorize them or pick a specific word like MESSAGE same as above themselves URL...